index.html

```diff
@@ -231,6 +231,22 @@
 
 
     // HISTOGRAMS
```

Comment:
Just to be a little more helpful here, you'll find that `bins` might not be quite finished yet. Remember to work incrementally, testing your assumptions about what each thing in your code does before moving onto the next step.

Also, remember this note in the assignment:

> Be sure to read the documentation on `histogram.value()`.